packages/@okta/vuepress-site/docs/guides/oauth-for-okta-apis/index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: OAuth for Okta APIs
```

Comment:
We should probably be consistent in how we refer to this. "OAuth 2 for Okta APIs" ?

We variously have
OAuth for Okta APIs
OAuth 2 for Okta APIs
OAuth 2.0 ... 
etc.